README.md

```diff
@@ -103,7 +103,7 @@ sudo flatpak override --env=GTK_THEME="catppuccin-${FLAVOR}-${ACCENT}-standard+d
 
 ### Manual installation
 
-If your distro does not package our theme, and the installation script will not work for your use case, you can pull down releases and extract them yourself. You can find the [latest release on GitHub](https://github.com/catppuccin/gtk/releases/latest).
```

Comment:
Hmm the first change is good since there could be unreleased commits on main, but the existing link here already goes to the latest *release*?

Reply:
Well the idea is that the tag would be the latest release, no?

Reply:
The benefit is that you'd be able to go into commit history and actually have the latest release point to the latest release _at that point in time_. I think it's nice :thinking: